src/ra_server.hrl

```diff
@@ -16,6 +16,7 @@
          uid :: ra_uid(),
          log_id :: unicode:chardata(),
          metrics_key :: term(),
+         metrics_labels :: map(),
```

Comment:
this can default to the empty map to save some defaulting later in tests and that.

Also is `map()` the most precise type we can use here?